src/styles/app.css

```diff
@@ -3,3 +3,76 @@ body {
     padding: 0;
     margin: 0;
 }
+
+div.ant-modal-content {
+    border-radius: 30px;
```

Comment:
Let's try to move CSS to inline as much as possible and avoid css files.

If you need css, use styled-components